src/AddressRegistry.Infrastructure/Modules/EditModule.cs

```diff
@@ -33,11 +31,9 @@ public class EditModule : Module
 
         protected override void Load(ContainerBuilder builder)
         {
-            var eventSerializerSettings = EventsJsonSerializerSettingsProvider.CreateSerializerSettings();
+            builder.RegisterModule(new CommandHandlingModule(_configuration));
 
-            builder
-                .RegisterModule(new EventHandlingModule(typeof(DomainAssemblyMarker).Assembly, eventSerializerSettings))
-                .RegisterModule(new CommandHandlingModule(_configuration));
+            builder.RegisterSnapshotModule(_configuration);
```

Comment:
I think `RegisterSnapshotModule` should be moved to the` AggregateSourceModul`e. It was previously registered in the `CommandHandlingModule`. And the `CommandHandlingModule` is only referenced by the `EditModule`.

Reply:
`StreetNameRegistry` does not have snapshotting in aggregatesourcemodule. Since it's not really needed for aggregatesource to function.

Reply:
And here in `AddressRegistry`?